contact.html

```diff
@@ -74,15 +75,14 @@
                 </div>
             </div>
         </section>
-        <footer>
+        <div class= "footer footer-fixed-bottom">
             <div class="container">
                 <div class="col-md-3">
                     <h3>CONTACT US</h3>
                     <p>
-                        Jatin Patil<br/>
-                        (Project Director)
+                        James Oswald<br/>
```

Comment:
Consider adding the (Managing Director) here, or otherwise remove the `<br/>`. It makes an awkward line break.